packages/cli-plugin-scaffold-react-component/.babelrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("../../.babel.node")({ path: __dirname });
```

Comment:
This needs to be `babel.react`, not `babel.node`.

Reply:
This should be babel.node since it is for building the scaffold.